yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -214,7 +214,7 @@ main() {
 
                 log "INFO" "argument pgss_enabled=$pgss_enabled"
                 if [[ "$pgss_enabled" == "false" ]]; then
-                    print_and_log "WARN" "Skipping $script_action: argument pgss_enabled is set as false"
+                    print_and_log "WARN" "Skipping $script_action: pg_stat_statements are not enabled"
```

Comment:
It should be “is” because pg_stat_statements refers to a singular PostgreSQL extension.